src/collectd.conf.pod

```diff
@@ -5121,6 +5121,36 @@ setting is given, the B<Address> and B<Port> settings are ignored.
 
 =back
 
+=head2 Plugin C<mesh>
+
+The I<Mesh plugin> collects some statistics about the Wifi Mesh performance.
+The Linux kernel requires to be compile with CONFIG_MAC80211_DEBUGFS=y.
```

Comment:
You could mention that as data is in debugfs, `collectd` would need to be run as `root` to access.  Maybe also that doing that with network facing service is a bit risky...